package.json

```diff
@@ -40,6 +41,7 @@
     "png-chunks-encode": "1.0.0",
     "png-chunks-extract": "1.0.0",
     "points-on-curve": "0.2.0",
+    "porter-stemmer": "0.9.1",
```

Comment:
this package is not used so can be removed from this PR. 
Also, this package is not maintained (the last publish was 10 years ago) so still do we want to use it? 
Instead, we can use [stemmer](https://www.npmjs.com/package/stemmer), well maintained with good usage.

Reply:
good call – I removed the package and will use the suggested one for next PR